### PR TITLE
[Accessibility] Fix misspelling in plugins/woocommerce-blocks/docs

### PR DESCRIPTION
**Pull request by Manussakis**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Fixes part of the [Misspelling all over the project](https://github.com/woocommerce/woocommerce/issues/37502) issue.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install typos CLI: https://github.com/crate-ci/typos.
2. Run `typos` in the `woocommerce/plugins/woocommerce-blocks/docs` folder.
3. Verify there are no typos to be fixed.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

**Comment by Manussakis** [comment]
@dkotter This PR is ready for code review.
Please, [check this comment for more context](https://github.com/woocommerce/woocommerce/pull/48413#pullrequestreview-2123538663).

File: plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md [hunk 1]
```diff
@@ -211,12 +211,12 @@ Here is a [quick demo](https://github.com/woocommerce/woocommerce-blocks/assets/
...
 #### Default template
 
-1. Go to Editor and edit "Single Product" template (Producto individual)
+1. Go to Editor and edit "Single Product" template (Product individual)
 2. Check the Related Products heading is translated and has top and bottom margin
 
 #### Transform from classic template
 
-1. Go to Editor and edit "Single Product" template (Producto individual)
+1. Go to Editor and edit "Single Product" template (Product individual)
```
Comment (albarin) [inline plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md]: I think the two changes in this file (plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md) should be reverted, they are not typos but text in Spanish.

**Review by dkotter — APPROVED** [review]
One comment about text that is in Spanish and thus not a typo but otherwise this looks good to me

File: plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md (hunk 1 above)
Comment (Manussakis) [reply plugins/woocommerce-blocks/docs/internal-developers/testing/releases/1160.md]: Thanks @albarin for catching that!
Fixed in:
- [Revert change of "Producto" word](https://github.com/woocommerce/woocommerce/pull/48562/commits/4e3d93d908423a8b0937091a3181020d39617d30)
- [Revert change of "Producto" word](https://github.com/woocommerce/woocommerce/pull/48562/commits/3d460ef411177146671c99ab14690db4e140e4e4)

**Review by dkotter — APPROVED** [review]

**Comment by dkotter** [comment]
@Manussakis This looks good to me and I don't think any need for QA on this one, so this is ready to be opened up and move to Woo review

**Comment by Manussakis** [comment]
@vikrampm1 This PR is ready to be reviewed by the appropriate Woo team.

**Review by danielwrobert — APPROVED** [review]
All looks good and I'm going to get this merged now – thanks for the contribution! 🎉

**Pull request merged**